src/components/Displaycard.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import Card from "./Card";
+import AddTodo from "./AddTodo";
+import UpadateForm from "./UpadateForm";
+import {
+  fetchTodos,
+  deleteTodo,
+  deleteMultipleTodos,
+} from "../store/todoSlice";
+
+function Displaycard() {
+  const dispatch = useDispatch();
+  const todos = useSelector((state) => state.todos.todos);
```

Comment:
@Jyoti657 ideal way should be like this
```suggestion
  const todos = useSelector((state) => state.todos.data);
```

Reply:
I will make a change in the project name with a meaningful word and not  repeat same words.